locales/enUS/translation.json

```diff
@@ -155,6 +155,16 @@
     "syncMessage": "Balance may be out-of-date while wallet syncs with ledger."
   },
 
+  "CrashReportModal": {
+    "uhOh": "Uh Oh!",
+    "encounteredAnIssue": "It seems we've encountered an issue that's causing the app the crash!",
+    "sendReport": "It would be super helpful if you sent the crash report to us, but we totally understand if you'd rather not :)",
+    "sendDebugLog": "If you'd like to also send us a copy of the debug log, you can find a copy of it",
+    "here": "here",
+    "sendDebugLog2": "so feel free to check it out.",
```

Comment:
Shouldn't wrapping be left to HTML/CSS, instead of doing it by hand?

Reply:
This isn't for wrapping, it's to apply a different styling to "here" (in this case it's a clickable button that exists in the line of text)